Update gps-tracker.md

mlagally opened the pull request.

mmccool commented:
I have merged your changes here into my PR on smartcity-localization (the name was also changed).  Please let's not edit the same files in two different PRs, so I suggest we close this PR and focus on the other one.

mlagally commented:
> I have merged your changes here into my PR (#468) on smartcity-localization (the name was also changed). Please let's not edit the same files in two different PRs, so I suggest we close this PR and focus on the other one. I updated the title of the other PR to reflect the fact it was also updating the localization use-case (which I'm about to rename yet again to "geolocation").

I did not notice that your other PR includes also changes to the geolocation use case, since the title only mentioned healthcare.
Merging my PR into your PR is ok to resolve the immediate issue, however please let's make sure that PRs are focused on single use cases, if possible.

mlagally commented:
Replaced by https://github.com/w3c/wot-architecture/pull/485
